doc/doc-site/docs/off-chain/python-sdk.md

```diff
@@ -1,3 +1,262 @@
+# deploy the faucet (all one command)
+aptos move publish \
+        --override-size-check \
+        --included-artifacts none \
+        --named-addresses econia=$ECONIA_ADDR \
+        --profile econia_exchange_deploy \
+        --assume-yes
+```
+
+It's to run the script! Setting our environment variables will have cleared the initial setup prompts for us. In order to run, install Poetry then install dependencies and run the script:
+
+```bash
+curl -sSL https://install.python-poetry.org | python3 - # only if necessary
+cd ./econia/src/python/sdk && poetry install # only if necessary
+poetry run trade # we're off to the races!
+```
+
+### Understanding the Example Script
+
+#### Step #1: Set-up the Market
+
+```
+Press enter to initialize (or obtain) the market.
+Market does not exist yet, creating one...
+EVENT SUMMARY: MarketRegistrationEvent
+  * Base Type (unit of lots): 0x...::test_eth::TestETH
+  * Quote Type (unit of ticks): 0x...::test_usdc::TestUSDC
+Market ID: 1
+TRANSACTIONS EXECUTED (first-to-last):
+  * Create a new market: 0x544c1535d9634bb4d4f5ee5b74ae798e45e5b1a0e54bfea060026e5ce0163dd9
+There are no open orders on this market right now.
+```
+
+In this case, the market did not exist for the pair of faucet currencies (`TestEth` and `TestUSDC`) that we deployed. Only one market can exist per unique configuration so once it's created here, it won't be created again in future runs. Instead it'll be detected as existing via a view function, and then re-used from thereon. There don't appear to be any open orders on this brand-new market which makes a lot of sense!
+
+#### Step #2: Set-up the Account "A"
+
+```
+Press enter to set-up an Account A with funds.
+New market account after deposit:
+  * tETH: 0 -> 10.0
+  * tUSDC: 0 -> 10000.0
+Account A was set-up: 0xb5a2e0363ecfd9fbc9cd9e77f5db5d7d6e85a1c8b931675f71e99a1836adfe41
+TRANSACTIONS EXECUTED (first-to-last):
+  * Mint 10.0 tETH (yet to be deposited): 0xbecafd3453798384eacdc25733d53e30d8d86524c27cf2f8da533a3ad785838a
+  * Mint 10000.0 tUSDC (yet to be deposited): 0xfdddda8daf3881908c413ec5d8cf12fb3a2d164d57a43908b06236171cfd1103
+  * Register a new account in market 1: 0x89a403548b2b986555751fa0e4d2d29e6a099ca38dd2d5640297b587fb27510c
+  * Deposit 10.0 tETH to market account: 0x8233bc577deafd9f3ace59c9d7843807c1a41bf9604e980eddff47990d677bdc
+  * Deposit 10000.0 tETH to market account: 0xd6b6eeb06160c13c174ea4c868fef9457686f45c7839463640cb7a5b051f95a3
+```
+
+Interacting with the exchange as an address requires a market account for each trading pair. Market accounts possess the funds available to trade on a trading pair as an account. In the above, we perform two steps for two coins, in addition to registering a new market account:
+
+1. Create a new market account (only if necessary).
+2. Obtain the funds to be deposited (in this case, we mint them).
+3. Depost the funds into the account.
+
+This gives us the 5 total transactions we expect and see above.
```

Comment:
Can you clarify that mint and deposit happens twice? When I first read this I was confused because there were 3 steps but 5 txns mentioned